README.md

```diff
@@ -161,6 +161,7 @@ export default [
   like integers but are too large, and
   [subnormal numbers](https://en.wikipedia.org/wiki/Subnormal_number).
 - `no-unnormalized-keys` - warns on keys containing [unnormalized characters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#description). You can optionally specify the normalization form via `{ form: "form_name" }`, where `form_name` can be any of `"NFC"`, `"NFD"`, `"NFKC"`, or `"NFKD"`.
+- `sort-keys` - warns when keys are not in the specified order. Based on [eslint/sort-keys](https://eslint.org/docs/latest/rules/sort-keys).
```

Comment:
```suggestion
- `sort-keys` - warns when keys are not in the specified order. Based on the ESLint [`sort-keys`](https://eslint.org/docs/latest/rules/sort-keys) rule.
```

Reply:
@nzakas Updated in https://github.com/eslint/json/pull/76/commits/53b4cdaf1fc55e6293d41f0eb04827e7ae76e3ed; apologies if GitHub  already sent a notification. Thanks!